### PR TITLE
Use Maven for builds and dependency management

### PR DESCRIPTION
**Pull request by rabeyta**

### CODE REVIEW

File: mutation-runner-plugin/src/main/java/sf/mr/ConsoleManager.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+public class ConsoleManager {
+    
+    private static final String CONSOLE_NAME = "MutationRunnerConsole";
+    
+    public static void printExceptionToConsole(MessageConsoleStream consoleStream, Exception exception) {
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        PrintStream outputPrintStream = new PrintStream(baos);
+        exception.printStackTrace(outputPrintStream);
+        consoleStream.print("\n");
+        consoleStream.print(exception.getMessage());
```
Comment (DylanLWScott) [inline mutation-runner-plugin/src/main/java/sf/mr/ConsoleManager.java]: MInor: Why didn't we just use consoleStream.println?

**Comment by DylanLWScott** [comment]
And for version number, did we want 1.2.2?

File: mutation-runner-plugin/src/main/java/sf/mr/ConsoleManager.java (hunk 1 above)
Comment (rabeyta) [inline mutation-runner-plugin/src/main/java/sf/mr/ConsoleManager.java]: No idea lol. will change.

**Comment by rabeyta** [comment]
@DylanLWScott Good call. We talked about making that 1.0.0-SNAPSHOT, but I forgot to go back and make that change. I'll update it.

**Comment by DylanLWScott** [comment]
Agree with changes :+1:

**Pull request merged**